src/components/Stepper.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="flex flex-col items-start gap-[8px]">
+    <div class="flex-row justify-start hidden laptop:flex">
+      <IconStepDot
+        v-for="(item, index) in totalStep"
+        :key="item"
+        :class="[
+          'mr-[16px]',
+          step === index + 1 ? 'text-like-cyan' : 'text-shade-gray',
+        ]"
+      />
+    </div>
+    <span class="text-medium-gray text-[14px]">{{
+      `Step ${step} / ${totalStep}`
```

Comment:
Should translate this?

Reply:
[🌐 Add i18n to stepper & error status](https://github.com/likecoin/liker-land/pull/1588/commits/a2593a3add3ffc223c782c2fcdae8b0cf8a2eb4e)